src/components/legend/attributes.js

```diff
@@ -74,6 +74,19 @@ module.exports = {
             'Sets the amount of vertical space (in px) between legend groups.'
         ].join(' ')
     },
+    entrywidth: {
+        valType: 'number',
+        min: 0,
```

Comment:
I think we should provide a `dflt` for this attribute.
However using zero looks confusing.
@alexcjohnson

Reply:
I actually think it's nice to use zero to mean "automatic - use the text width," if we document that.

Alternatively, we could have a third value for `entrywidthmode`, perhaps `'auto'`, that means we use the text width. Then the logic would be:
- If a nonzero `entrywidth` is provided, `entrywidthmode` defaults to `'pixel'`. If not, it defaults to `'auto'`.
- If `entrywidthmode` is `'auto'` we don't coerce `entrywidth`.

That feels unnecessarily cumbersome to me though. Only reason I can think of that would justify this is if we want to actually support a width of zero. Any reason someone would want that?

Reply:
If we want to use zero as the auto default the `min` namely for the `pixel` mode should be a value greater or equal 1 not zero IMHO.

Reply:
That would be confusing, wouldn't it? We'd document "Use 0 to size the entry based on the text width" but document a min of 1... yet if you explicitly set it to 0 it would work, because we'd throw out the 0 and replace it with the default 0.

Let's keep this simple, and consistent with `colorbar.len` and `colorbar.thickness` - `min: 0` and no `max` regardless of `entrywidthmode`, the only difference being that `0` has a special meaning we'll document. If it turns out that certain values (either very small or very large) cause problems worse than "it looks bad," we can address that during the drawing process when we know more than we do at the supplyDefaults step.